src/server/game/Entities/Unit/Unit.h

```diff
@@ -757,6 +757,18 @@ struct CleanDamage
 struct CalcDamageInfo;
 struct SpellNonMeleeDamage;
 
+struct DelayedDamage
+{
+    Unit * attacker;
+    Unit * victim;
```

Comment:
Connect the \* to the pointer

Reply:
Will make the adjustment and test before pushing.

Reply:
https://github.com/azerothcore/azerothcore-wotlk/pull/14199/commits/bc3ea4d68565d7668c82e93aa538db6c36c770c6